src/oc_erchef/apps/oc_chef_wm/src/chef_wm_search.erl

```diff
@@ -410,40 +396,15 @@ encode_results(Results, Prefix, Acc) ->
 %% Encode a list of items as a JSON array partial. That is, encode as a JSON array and then
 %% strip the '[' and ']' off the result. This allows us to incrementally encode a long array
 %% of objects in batches avoiding having all objects in memory in order to encode them.
-%%
-%% This function knows how to deal with gzip binary from SQL and with EJSON data coming
-%% straight from couch. If the data has come from couch, this is where couch cruft keys _id
-%% and _rev are removed.
 encode_result_rows([Item|_Rest]=Items) when is_binary(Item) ->
     ItemList = << <<(chef_db_compression:decompress(Bin))/binary, ",">> || Bin <- Items >>,
     %% remove trailing "," from binary
     binary:part(ItemList, {0, size(ItemList) - 1});
 encode_result_rows(Items) ->
-    %% ensure no couchdb cruft leaks out
-    CleanItems = [ {remove_couchdb_keys(Doc)} || {Doc} <- Items ],
```

Comment:
As with the other change, I had left this one in in my original branch.  I'm less sure about removing it - is it possible that any of these keys remain in migrated data?